client/curriculum/html/LinksAndButtons.tsx

```diff
@@ -0,0 +1,882 @@
+              Analytics) să identifice pagina ca sursă a link-ului pe care s-a
+              dat click.
+            </li>
+          </ul>
+        </section>
+        <section>
+          <LessonHeading as="h3" id="intrebari-bonus">
+            Întrebări BONUS
+          </LessonHeading>
+          <p>
+            În continuare vom raspunde la câteva întrebări pe care noi ca și
+            developeri ni le putem adresa pe măsură ce codăm și ne întâlnim cu
+            diverse situații:
+          </p>
+          <blockquote>
+            Ar trebui să folosim atributul
+            {' '}
+            <strong className="formatted">title</strong>
+            {' '}
+            împreuna cu tag-ul a?
+          </blockquote>
+          <p>
+            Hmmm..probabil că nu. În momentul în care mergem cu mouse-ul în
+            dreptul elementului (facem
+            {' '}
+            <i>hover</i>
+            ) ce are ca și atribut title
+            vom vedea un pop-up cu textul pe care l-am scris.
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a title="Nu ar trebui să fiu aici" href="/">
+  Un simplu exemplu
+</a>`}
+          />
+          <LessonFigure
+            withBorder
+            src="/images/lessons/links-and-buttons/title-link.png"
+            alt="Atributul title ca link"
+            demo="/demo/html/atributul-title-ca-link"
+          />
+          <p>
+            Acest pop-up nu este atât de accesibil și nu poate fi stilizat. Plus
+            că pe device-urile cu touch este imposibil de folosit. Trebuie să ne
+            asigurăm întotdeauna că oferim un text suficient de descriptiv
+            pentru a ști la ce să ne așteptăm când dăm click pe un link.
+          </p>
+          <blockquote>
+            Putem folosi doar un
+            {' '}
+            <strong className="formatted">icon</strong>
+            {' '}
+            în
+            interiorul unui link?
+          </blockquote>
+          <p>
+            Așa cum am precizat deja, un simplu icon nu ne-ar oferi suficientă
+            informație contextuală despre link, deci nu ar fi chiar recomandat.
+            Dacă totuși suntem nevoiți să folosim un astfel de icon, o bună
+            practică ar fi să utilizam un pattern de genul în care adăugam niște
+            text ascuns ce va oferi context pentru screen readere:
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="/">
+  <!-- Ascundem icon-ul de assistive technology -->
+  <svg viewBox="0 0 32 32" width="32px" height="32px" 
+  aria-hidden="true" focusable="false">
+  <circle cx="16" cy="16" r="16"/>
+  </svg>
+  <!--Se comporta ca un label ce e ascuns -->
+  <span style="position: absolute; opacity: 0">
+  Useful link text
+  </span>
+ </a>`}
+          />
+          <LessonFigure
```

Comment:
Inline demo. I think we can keep the demo page, this is actually a little more interesting.